### DISCUSSION TITLE
Is it possible to remove tracking in form of "sub-elements" in URLs?

### DISCUSSION THREAD

**Discussion by Tiagoquix** [uBlockOrigin/uBlock-issues#2213] [Q&A] [answered]
For example, whenever I click the Amazon logo in their website it returns this:

`https://www.amazon.com/ref=nav_logo`

Would it be possible, using any of uBlock's functionality (filter syntax etc.), to remove the `ref=nav_logo`?

I tried `$removeparam` but it doesn't work, since it isn't `?ref=nav_logo` (i.e. doesn't start with a question mark).

**Comment by gothic-bum** [comment, marked as answer]
Currently not possible. 

uBO is only capable of removing parameters from [queries](https://en.wikipedia.org/wiki/Query_string) (i.e. `?` or `&` in URL).
